src/scanner.ts

```diff
@@ -18,10 +18,12 @@ import {
   init,
   finishBatch,
 } from './store'
-import interpretFile from './interpreter'
+import interpretFile, { interpretBallotString } from './interpreter'
 
 import exec from './exec'
 
+let manualBatchId: number | undefined
```

Comment:
Feels like there must be a way to do this without using a `let` and having to reset to `undefined`, but okay for now.